readme.md

```diff
@@ -48,7 +48,17 @@ let add_numbers = function( a, b ) {
 	
 }
 ```
-Functions can return a singular variable using the `return` keyword. Do note that any code after a return is unreachable and will be ignored by the implementation. As such, the following is **invalid** syntax:
+So is this:
+```
+func add_numbers ( a, b ) {
+	
+	let c = a + b;
+	return c;
+	
+}
+let add_numbers_copy = add_numbers;
```

Comment:
I feel like this should be `let add_numbers_copy = add_numbers(a,b)`?

Reply:
But it is showing that variables can be assigned to functions.

Reply:
That’s kinda a weird way to do that, No? But alrighty

Reply:
> That’s kinda a weird way to do that, No? 

I don't really understand, what would be a better way to do it?